### DISCUSSION TITLE
Feedback button disappearing when I click or mouseover

### DISCUSSION THREAD

**Discussion by fatih-erikli** [community/community#19698] [Other Feature Feedback, Questions, & Ideas]
Link: https://github.com/Azure-Samples/AzureMapsCodeSamples/blob/master/Static/data/geojson/GpsTrace.json
![feedback](https://user-images.githubusercontent.com/92447202/176476414-8df1bde0-9f69-46f9-b641-9c4e25f4ee69.gif)

**Comment by benjiallen** [comment]
I don't think this is a GitHub issue. GitHub is just rendering content within the repo. I suggest you contact the owner of the repo and leave feedback within that repo.

**Reply by fatih-erikli** [reply]
I don't think repository owner can do anything. This is either github or bing maps issue.
